### DISCUSSION TITLE
Support Zig v14

### DISCUSSION THREAD

**Discussion by yonas** [ghostty-org/ghostty#3590] [Feature Requests, Ideas] [answered]
> build.zig:38:9: error: Your Zig version v0.14.0-dev.2569+30169d1d2 does not meet the required build version of v0.13.0

**Comment by jcollie** [comment]
This will happen sometime after Zig 0.14 is released.

**Comment by mitchellh** [comment, marked as answer]
We only support the latest released Zig and only some time after its released.

The reason is because we do not own packaging, distros do, and distros cannot package Ghostty unless all dependencies are also packaged. If we switch to new updates too quickly, Ghostty can't be packaged.

We must be patient.

**Reply by mitchellh** [reply]
As an updated note, since Zig 0.14 has been officially released, Ghostty has moved to Zig 0.14 for releases after the 1.1.x series (the 1.1.x series will remain on 0.13 for any backports). Main branch is on Zig 0.14. This happened some weeks ago.
